src/components/pinCodes/SSIPinCode/index.tsx

```diff
@@ -76,7 +76,7 @@ class SSIPinCode extends PureComponent<IProps, IState> {
     const {onVerification} = this.props;
 
     onVerification(value)
-      .then(() => this.setState({retry: 0, pin: ''}))
+      .then(() => this.setState({retry: 0, pin: value}))
```

Comment:
the pin in the state should already contain the value. so setting it again is redundant.

Reply:
if by 'more input' you mean adding text to the pin-code text box, it wasn't the issue, the issue was since we're setting it to an empty value, the empty screen was showed again, and you could type in; and of course it closes immediately